Dockerfile

```diff
@@ -23,27 +25,71 @@ RUN apt-get update && \
     apt-get install -y git-lfs graphviz libgl1 && \
     git lfs install
 
+# install vtk
+COPY . /modulus-sym/
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ] && [ -e "/modulus-sym/deps/vtk-9.2.6.dev0-cp310-cp310-linux_aarch64.whl" ]; then \
+	echo "VTK wheel for $TARGETPLATFORM exists, installing!" && \
+	pip install /modulus-sym/deps/vtk-9.2.6.dev0-cp310-cp310-linux_aarch64.whl; \
```

Comment:
Could you please explain why you are using this pip wheel of vtk instead of direct pip installation?

Reply:
Please refer my comment here: https://github.com/NVIDIA/modulus-sym/pull/54#discussion_r1334636520

Reply:
The wheel is basically a pre-build wheel that we can use for faster build if present (helps for local builds). But if not present, it reverts to the build from source (for arm)